Docker-compose.yml

```
@@ -0,0 +1,24 @@
+version: "3"
+
+services:
+  Development:
+    env_file:
+        - .env
+    build: . 
+    
+    .volumes: 
+        - type: bind
+          source: /todo_app
+          target: /app
```

```suggestion
          source: ./todo_app
          target: /todo_app/todo_app
```

Don't forget the `.` to mean relative to the current directory and you want to keep the file structure - python files in a todo_app subfolder. This copies the contents of the source folder into the contents of the target folder. So you need to keep the second `/todo_app` 

Not sure where `/app` came from - you set WORKDIR to `/todo_app` in your Dockerfile.